pkg/term/term_linux_test.go

```diff
@@ -3,29 +3,20 @@
 package term
 
 import (
-	"flag"
 	"io/ioutil"
 	"os"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-var rootEnabled bool
-
-func init() {
-	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
These tests work with the terminal, so any failure to cleanup should be isolated to the terminal. This patch also includes a fix for a test that did not cleanup properly (`defer RestoreTerminal()`).